src/controllers/utils/types.ts

```diff
@@ -28,3 +29,11 @@ export type Solution = {
 };
 
 export type TopSolution = { solution: Solution; metaData: Meta };
+
+export type Regex = SearchRegExp;
```

Comment:
why are you re-exporting `Regex` when you could use `SearchRegExp` directly?

Reply:
Oh, this was an oversight, I initially had an extra typed field joined to SearchRegExp. I'll remove the Regex type